python/tests/unit/arcticdb/version_store/test_normalization.py

```diff
@@ -36,7 +37,8 @@
     NPDDataFrame,
 )
 from arcticdb.version_store._common import TimeFrame
-from arcticdb.util.test import param_dict, CustomThing, TestCustomNormalizer, assert_frame_equal, assert_series_equal
+from arcticdb.util.test import \
+    param_dict, CustomThing, TestCustomNormalizer, assert_frame_equal, assert_series_equal, IS_PANDAS_ZERO
```

Comment:
Not a thing in `black`

Reply:
Not going to attempt manual Black formatting or running the auto-formatter till it's in the CI